src/build-all-functions-definitions.ts

```diff
@@ -0,0 +1,49 @@
+import { FunctionDefinition } from "./meta-function-type";
+import { getDescriptorFileContent } from "./get-file";
+import { isFunctionDefinition } from "./is-function-definition";
+
+export async function buildAllFunctionDefinitions (
+  definitionsAndPaths : Array<string | FunctionDefinition>,
+  workingDir = "./",
```

Comment:
I believe simply using `""` here is better because Windows. Also the paths "path/to/file.ext" and "./path/to/file.ext" are equivalent.

Reply:
good to know, I will replace all the occurrences of this case then.